js/ui/diagram/ui.diagram.commands.js

```diff
@@ -0,0 +1,99 @@
+                icon: "undo"
+            },
+            {
+                name: DiagramCommand.Redo,
+                hint: "Redo",
+                icon: "redo"
+            },
+            {
+                name: DiagramCommand.FontName,
+                beginGroup: true,
+                widget: "dxSelectBox",
+                items: ["Arial", "Arial Black", "Helvetica", "Times New Roman", "Courier New", "Courier", "Verdana", "Georgia", "Comic Sans MS", "Trebuchet MS"]
+            },
+            {
+                name: DiagramCommand.FontSize,
+                widget: "dxSelectBox",
+                items: ["8pt", "9pt", "10pt", "11pt", "12pt", "14pt", "16pt", "18pt", "20pt", "22pt", "24pt", "26pt", "28pt", "36pt", "48pt", "72pt"]
+            },
+            {
+                name: DiagramCommand.Bold,
+                hint: "Bold",
+                icon: "bold"
+            },
+            {
+                name: DiagramCommand.Italic,
+                hint: "Italic",
+                icon: "italic"
+            },
+            {
+                name: DiagramCommand.Underline,
+                hint: "Underline",
+                icon: "underline"
+            },
+            {
+                name: DiagramCommand.FontColor,
+                text: "Text Color",
+                widget: "dxColorBox"
+            },
+            {
+                name: DiagramCommand.StrokeColor,
+                text: "Line Color",
+                widget: "dxColorBox"
+            },
+            {
+                name: DiagramCommand.TextLeftAlign,
+                hint: "Align Left",
+                icon: "alignleft",
+                beginGroup: true
+            },
+            {
+                name: DiagramCommand.TextCenterAlign,
+                hint: "Center",
+                icon: "aligncenter"
+            },
+            {
+                name: DiagramCommand.TextRightAlign,
+                hint: "Align Right",
+                icon: "alignright"
+            },
+            {
+                name: DiagramCommand.ConnectorLineOption,
+                widget: "dxSelectBox",
+                text: "Line Option",
+                items: ["Straight", "Orthogonal"]
+            },
+            {
+                name: DiagramCommand.ConnectorStartLineEnding,
+                widget: "dxSelectBox",
+                text: "Start Line Ending",
+                items: ["None", "Arrow"]
+            },
+            {
+                name: DiagramCommand.ConnectorEndLineEnding,
+                widget: "dxSelectBox",
+                text: "End Line Ending",
+                items: ["None", "Arrow"]
+            },
+            {
+                name: DiagramCommand.AutoLayoutTree,
+                text: "Tree Auto Layout"
+            },
+            {
+                name: DiagramCommand.AutoLayoutTree,
+                text: "Layered Auto Layout"
+            }
+        ];
+    }
+};
+
+module.exports = DiagramCommands;
```

Comment:
doesn't work - tests fail. see https://devextreme-ci.devexpress.com/DevExpress/DevExtreme/14779
Is this a must-have change?

Reply:
`DevExpress.ui[componentName].subclassOf is not a function`

Hm...
@DokaRus @jsdmitry are you sure that we can use ES6 classes for widget inheritance instead of DX.Class.inherit?